phys2bids/phys2bids.py

```diff
@@ -35,6 +35,7 @@
 
 from phys2bids import utils, viz, _version
 from phys2bids.bids import bidsify_units, use_heuristic, participants_file
+from phys2bids.bids import dataset_description_file, readme_file
```

Comment:
At this point, it would be better to:
```suggestion
from phys2bids import utils, viz, _version, bids
```
And then adapt the rest of the code